test/function/samples/max-parallel-file-reads-unlimited-zero/_config.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert');
```

Comment:
See my comment in the other test to make this one really test the actual file system activity.

Reply:
Done